src/services/HttpService.js

```diff
@@ -57,7 +64,7 @@ export class HttpService {
 	 */
 	async post(resource, data, contentType, options = {}) {
 		const fetchOptions = {
-			mode: 'cors',
+			mode: HttpService.DefaultRequestMode,
 			method: 'POST',
```

Comment:
To be consequent, the request method-types could also be extracted to private CONST

Reply:
Need more explanation...

Reply:
Probably I should name the static getter `DEFAULT_REQUEST_MODE` to indicate it has public access?